src/consumers/consumers.h

```diff
@@ -1,6 +1,84 @@
 #ifndef CONSUMERS_
 #define CONSUMERS_
 
+typedef struct metric_metadata {
+    tag_collection* tags;
+    char* instance;
+    char* sampling;
+} metric_metadata;
+
+typedef struct counter_metric {
+    char* name;
+    metric_metadata* meta;
+    unsigned long long int value;
+} counter_metric;
+
+typedef struct counter_metric_collection {
+    counter_metric** values;
+    long int length;
+} counter_metric_collection;
+
+typedef struct gauge_metric {
+    char* name;
+    metric_metadata* meta;
+    signed long long int value;
```

Comment:
Gauge should be float or double, probably the latter one. It's basically an arbitrary (float) number.

Reply:
Gauge is double by 3363499